soda/core/tests/helpers/common_test_tables.py

```diff
@@ -108,3 +108,35 @@
     columns=customers_test_table.columns,
     values=customers_test_table.values,
 )
+
+customers_profiling = TestTable(
```

Comment:
Why this variation?  Why can't the existing test table not be used for this?

Reply:
it presented some annoying ties between some records and lead to differences in how ordering would be resolved between local and remote dbs although they are the same as well as some painful arbitrary conversions of float into ints. 

So this variation prevent those from happening but is just valid a test as the other ones.